modules/gwlb/variables.tf

```diff
@@ -80,21 +159,47 @@ variable "backends" {
       }
     }
   }
-  type = map(any)
+  nullable = false
+  type = map(object({
+    name = string
+    tunnel_interfaces = map(object({
+      identifier = number
+      port       = number
+      protocol   = optional(string, "VXLAN")
+      type       = string
+    }))
+  }))
+  validation { # protocol
```

Comment:
if we can only have two backends, I would add a validation for that

Reply:
Added